server/graphql/v2/mutation/AgreementMutations.ts

```diff
@@ -119,14 +136,16 @@ export default {
 
       await twoFactorAuthLib.enforceForAccount(req, host);
 
-      const toUpdate: Parameters<AgreementModel['update']>[0] = {};
-
-      if (args.title) {
-        toUpdate.title = args.title;
-      }
-
-      if (args.expiresAt) {
-        toUpdate.expiresAt = args.expiresAt;
+      const toUpdate: Parameters<AgreementModel['update']>[0] = pick(args, ['notes', 'title', 'expiresAt']);
```

Comment:
Simplified a bit the approach by using `pick`. It provides a cleaner behavior out-of-the box:

```es6
// This will prevent setting `expiresAt` to null; should be `if (args.expiresAt !== undefined)`
if (args.expiresAt) {
  toUpdate.expiresAt = args.expiresAt;
}

// Anyway, it's simpler to just use pick as it won't pick the value if it's `undefined`
const toUpdate = pick(values, ['expiresAt']);
```